dataactbroker/app.py

```diff
@@ -103,7 +83,7 @@ def runApp():
 
 if __name__ == '__main__':
     runApp()
-    proc = multiprocessing.Process(target=checkDynamo)
+    proc = multiprocessing.Process()
```

Comment:
Paging @nmonga91 on this one as I'm not actually sure what these are doing.  This code is not used in the AWS version of the app or the unit tests, but should still be used by local installs.